lib/identity_cache/configuration_dsl.rb

```diff
@@ -42,18 +42,19 @@ module ClassMethods
       # * unique: if the index would only have unique values
       #
       def cache_index(*fields)
-        ensure_base_model
         raise NotImplementedError, "Cache indexes need an enabled primary index" unless primary_cache_index_enabled
         options = fields.extract_options!
-        self.cache_indexes.push fields
+        unique = options[:unique] || false
```

Comment:
ah, got it, it can't be `nil` otherwise will default to `true` in `cache_attribute`